src/utils.ts

```diff
@@ -109,6 +109,8 @@ export function transformDateInResponse<T>(data: T): T {
     'uploadedTime',
     'deletedTime',
     'timestamp',
+    'sourceCreatedTime',
+    'sourceModifiedTime',
```

Comment:
How do these two related to the topic?

Reply:
they are not really related, but they will go in a separate commit `feat(files): source created/modified time, rootAssetIds, assetSubtreeIds`, hence a separate item in the changelog as well.

Just wanted to save some time

Reply:
I see, than ready to merge